src/tests/test_gcl_page.py

```diff
@@ -249,7 +251,8 @@ def test_linked_in(browser):
 	page.load()
 	page.test_linked_in()
 	url = 'https://www.linkedin.com/company/girlscodelincoln'
-	assert browser.current_url == url
+	url2 = 'https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fgirlscodelincoln'
```

Comment:
Is there a `encodeURI()` type function in python that you can leverage here instead of hard coding?

Reply:
I have updated the code to now have an encoder. 

This has the same issue as above because linkedIn likes to know where you came from to get to linkedIn it will send a giant link first so they can track where the link originated. This link has a hash that changed every time you click it. Inside the container it never fully goes to the linked in site and fails when I wait for the page title. Copy and pasting the link works fine to redirect you outside the container. 

In other words linkedIn and docker containers are stupid. Which is why I did it the way I did.